src/Illuminate/Foundation/Console/stubs/console.stub

```diff
@@ -23,7 +23,7 @@ class {{ class }} extends Command
     /**
      * Execute the console command.
      */
-    public function handle()
+    public function handle(): void
```

Comment:
I worry that specifying void as a strict type, obfuscates the capability of using the handle function to return status codes, and without documentation could lead to confusion for future contributors (Making the assumption that a Command handle function will never return anything and can only ever be one of Laravel's 3 statuses `0 = SUCCESS`, `1 = FAILED`, `2 = NOT READY`).

For example, if you have the following function:

```php
public function handle()
{
    return 5;
}
```

If you run the artisan command, followed by `echo $?` to get the status code of the previously executed command, it would show 5. 

Do you have a thought how we could potentially handle this?

Reply:
It's just the default stub. 
After generating a new command, it can be changed by the developer without any effect on other parts of the application.
On the other hand, the examples in the document have this type declaration right now and with this PR there is no difference between the generated class and the document.